### DISCUSSION TITLE
Can't use Copilot in vscode after payment in monthly

### DISCUSSION THREAD

**Discussion by Las6040a** [community/community#30900] [Copilot Conversations] [answered]
In Vscode the copilot's state is  'Copilot activation failed' 

 How can i do ?

**Comment by davecheney** [comment, marked as answer]
@Las6040a Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by Las6040a** [reply]
Hi,davecheney
Thank your answer
I use copilot in home , Chain,I don't know are there any problems in SSL ,but i found that when i click the copilot icon in bottom of vscode
the state will change to 'activate copilot' ,is that a SSL problem?

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by Las6040a** [comment]
Github copilot:

nothing at all...... 


copilot authentication :

[Info  - 04:21:04.377] Reading sessions from keychain...
[Info  - 04:21:04.393] Getting sessions for user:email...
[Info  - 04:21:04.394] Getting sessions for all scopes...
[Trace  - 04:21:04.410] Token acquired from secret storage.
[Info  - 04:21:04.410] Got stored sessions!
[Trace  - 04:21:04.410] Read the following session from the keychain with the following scopes: read:user
[Trace  - 04:21:04.410] Read the following session from the keychain with the following scopes: user:email
[Info  - 04:21:04.410] Got 2 verified sessions.
[Info  - 04:21:04.412] Got 1 sessions for user:email...
[Info  - 04:21:04.412] Got 2 sessions for ...
[Info  - 04:21:10.3] Getting sessions for read:user...
[Info  - 04:21:10.3] Got 1 sessions for read:user...
[Info  - 04:46:10.821] Getting sessions for read:user...
[Info  - 04:46:10.821] Got 1 sessions for read:user...

**Reply by Las6040a** [reply]
Sorry, 

Github authentication:

[Info  - 04:21:04.377] Reading sessions from keychain...
[Info  - 04:21:04.393] Getting sessions for user:email...
[Info  - 04:21:04.394] Getting sessions for all scopes...
[Trace  - 04:21:04.410] Token acquired from secret storage.
[Info  - 04:21:04.410] Got stored sessions!
[Trace  - 04:21:04.410] Read the following session from the keychain with the following scopes: read:user
[Trace  - 04:21:04.410] Read the following session from the keychain with the following scopes: user:email
[Info  - 04:21:04.410] Got 2 verified sessions.
[Info  - 04:21:04.412] Got 1 sessions for user:email...
[Info  - 04:21:04.412] Got 2 sessions for ...
[Info  - 04:21:10.3] Getting sessions for read:user...
[Info  - 04:21:10.3] Got 1 sessions for read:user...
[Info  - 04:46:10.821] Getting sessions for read:user...
[Info  - 04:46:10.821] Got 1 sessions for read:user...

**Comment by davecheney** [comment]
@Las6040a looked at your telemetry I see no networking issues at the moment. 
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by Las6040a** [reply]
When select _Github Copilot_ from pulldown,The Output dispaly area is empty(nothing to display)

**Reply by davecheney** [reply]
@Las6040a sadly it looks like your traffic is being firewalled by your internet service provider, employer, or government.

**Reply by Las6040a** [reply]
Anyway,Thank you very much!!
